app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -69,30 +69,24 @@ class MainActivity : AppCompatActivity() {
 
         supportActionBar?.title = "Events"
 
-        val bundle = intent.extras
+        val bundle = if (savedInstanceState == null) intent.extras else null
 
-        if (savedInstanceState == null) {
-            loadFragment(supportFragmentManager, EventsFragment(), frameContainer.id)
```

Comment:
How does the new code handle this case?

Reply:
How does this case is used?, I don't understand. Because both issues are resolved with set bundle to null when orientation changed.